### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by vladislavshvets**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,18 @@
...
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine track = new Track();
 
+        Machine[] machines = new Machine[]{excavator, bulldozer, track};
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        for (Machine machine1 : machines) {
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/MainApp.java:25]:
```suggestion
        for (Machine machine : machines) {
```
i don't see any reasons to change naming in this loop

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -12,5 +12,18 @@
...
+            machine1.stopWork();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/MainApp.java:26]:
```suggestion
            machine.stopWork();
```

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (vladislavshvets) [reply src/main/java/core/basesyntax/MainApp.java]: yep, exactly, i didn't notice it, done

File: src/main/java/core/basesyntax/MainApp.java (hunk 2 above)
Comment (vladislavshvets) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**